src/ctap/storage.rs

```diff
@@ -211,6 +223,33 @@ impl PersistentStore {
                 })
                 .unwrap();
         }
+        if self.store.find_one(&Key::AttestationPrivateKey).is_none() {
```

Comment:
I introduced `USE_BATCH_ATTESTATION` for users to have a choice. Batch attestation can be a privacy concern, and some relying parties might not require it. If we want to replace the boolean flag `USE_BATCH_ATTESTATION` with "exists in the persistent store", we could add that here. This is more of an open question which is cleaner.

Reply:
I think it makes sense to do that for now (only copy the attestation to the storage if compiled with batch attestation and use batch attestation only if the attestation is present in the storage). It should not be hard to improve on that later (e.g. per-rpid batch attestation).